src/main/resources/assets/computercraft/lua/rom/motd.txt

```diff
@@ -28,7 +28,17 @@ Use "monitor" to run a program on a attached monitor.
 View all attached peripherals with "peripherals".
 Use "time" to see the in-game time.
 You can set the label of a computer with "label set".
-A computer needs a label to keep its files if it got destroyed.
+A computer needs a label to keep its files if it gets destroyed.
 You can disable auto completion in the shell with "set shell.autocomplete false".
 You can disable auto completion in edit with "set edit.autocomplete false".
 Feeling creative? Use a printer to print a book!
+Files beginning with a "." character are hidden from "list" by default.
```

Comment:
I wonder if it's worth merging this into the one about `list.show_hidden`. Or if that's too unwieldy.